index.js

```diff
@@ -21,7 +21,7 @@ const customConfig = {};
 
 const getConfig = function getConfig() {
   const args = [].slice.call(arguments);
-  return Object.assign.apply(null, [{}, defaults, customConfig].concat(args));
+  return _.merge.apply(_, [{}, defaults, customConfig].concat(args));
```

Comment:
Yes, that's literally the exact change I just made.

```
Object assign({ a: 1, b: 2 }, { c: 3}); // { c: 3 }
_.merge({ a: 1, b: 2 }, { c: 3}); // { a: 1, b:2, c: 3 }
```

In particular, it means we can set partial session properties from apps, without needing to specify the whole object, and therefore allows us to not have to re-implement the env var reading that bootstrap already does.